inst/assets/bibliography.bib

```diff
@@ -1908,3 +1908,286 @@ @article{yaverouglu2014revealing
 volume = {4},
 year = {2014}
 }
+
+
+
+
+
+
+
+
+
+@article{Lee2019,
+    author = {Lee YH},
+    title = {Strengths and Limitations of Meta-Analysis},
+    journal = {The Korean Journal of Internal Medicine},
+    volume = {94},
+    number = {5},
+    pages = {391-395},
+    url = {https://www.ekjm.org/journal/view.php?number=25636},
+    year = {2019}
+}
+
+
+@article{Finckh2008,
+    author = {Finckh A},
+    title = {Primer: strengths and weaknesses of meta-analysis},
+    journal = {Nature Clinical Practice Rheumatology},
+    volume = {4},
+    number = {3},
+    pages = {146-152},
+    url = {https://www.nature.com/articles/ncprheum0732},
+    year = {2008}
+}
+
+
+@article{Israel2011,
+    author = {Israel H},
+    title = {A guide to understanding meta-analysis},
+    journal = {Journal of Orthopaedic & Sports Physical Therapy},
+    volume = {41},
+    number = {7},
+    pages = {496-504},
+    url = {https://www.jospt.org/doi/10.2519/jospt.2011.3333?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed},
+    year = {2011}
+}
+
+
+@article{Hoffman2019,
+    author = {Hoffman JIE},
+    title = {Basic biostatistics for medical & biomedical practitioners},
+    journal = {Indian Journal of Medical Research},
+    volume = {154},
+    number = {6},
+    pages = {899–900},
+    url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9347243/},
+    year = {2019}
+}
+
+
+@article{Haidich2010,
+    author = {Haidich AB},
+    title = {Meta-analysis in medical research},
+    journal = {Hippokratia},
+    volume = {14},
+    number = {Suppl 1},
+    pages = {29-37},
+    url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049418/},
+    year = {2010}
+}
+
+
+@article{Ma2022,
+    author = {Ma S},
+    title = {Population structure discovery in meta-analyzed microbial communities and inflammatory bowel disease using MMUPHin},
+    journal = {Genome Biology},
+    volume = {23},
+    number = {1},
+    pages = {208},
+    url = {https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02753-4},
```

Comment:
could we have the (permanent) doi instead of url for all references where they are available?

Reply:
Updated!

Reply:
Resolve these discussions as they become ready.